src/env.cc

```diff
@@ -592,4 +602,25 @@ void Environment::AsyncHooks::grow_async_ids_stack() {
 
 uv_key_t Environment::thread_local_env = {};
 
+void Environment::Exit(int exit_code) {
+  if (is_main_thread())
+    exit(exit_code);
+  else
+    worker_context_->Exit(exit_code);
+}
+
+void Environment::stop_sub_worker_contexts() {
+  while (!sub_worker_contexts_.empty()) {
+    Worker* w = *sub_worker_contexts_.begin();
+    sub_worker_contexts_.erase(w);
```

Comment:
I think this line can replace with `remove_sub_worker_context(w)` like a `is_main_thread`.